README.md

```diff
@@ -65,6 +67,15 @@ bazel-bin/nighthawk_client  [--trace <uri format>]
 
 Where:
 
+--failure-predicate <<string, uint32_t>>  (accepted multiple times)
+Failure predicate. Allows specifying a counter name plus threshold
+value for failing execution. Defaults to not tolerating error status
+codes and connection errors.
+
+--termination-predicate <<string, uint32_t>>  (accepted multiple times)
+Termination predicate. Allows specifying a counter name plus threshold
+value for terminating execution.
```

Comment:
I wasn't familiar with the `<< >>` notation. What about `string:uint32_t`? 

Also could you add an example (`benchmark.http_4xx:50`) for both termination and failure predicate? I would have guessed `client.benchmark.http_4xx=50`.

Reply:
This notation originates from TCLAP .. (so we generate this fragment pre-commit via running `nighthawk_client --help`). Given that, wdyt? It would be possible to add a step in the format/fix step for this to massage it the way we want it to be, but I'm not sure it's worth it? (And then we'd still have the actual `--help` output using this notation).

As for an example, sure, I will add that. Its also easy to replace `:` with `=` at this stage, I agree that would be more intuitive.

Reply:
I see, if a tool is interpreting `<<>>` and the `:` isn't built in to `<<>>`, we should definitely keep `<<>>`.